contents/docs/user-guides/paths.mdx

```diff
@@ -6,26 +6,192 @@ showTitle: true
+
+### Understanding paths
+
+Paths are one of PostHog's most powerful tools for exploring how users journey through your app.
+However, they can be somewhat overwhelming when first getting started with them, simply due to the complexity and density of the visualization.
+This section provides a bit of background to help understand what paths are at a basic level, and how you can use them to discover things about your user journey.
+
+There are a couple of terms that will be handy to define beforehand:
+
+-   **Step** - This is a single point along a user's journey, and corresponds with the events that were sent during a user's session. For example, each page a user viewed or event they send will be a separate step.
+-   **Path** - This is the basic unit of the paths insight, and corresponds to a user _moving from one step to another_
+
+A step can be any [Event](/docs/user-guides/actions) or [Action](/docs/user-guides/actions) (such as pageviews, button clicks, or a combination of both).
+
+### Looking at a simple path
+
+Now that we know the basic concepts in a path, let's take a look at some simple examples.
+The basic concept of a path is most easily seen by setting the number of steps above the visualization to only 2.
+For simplicity, we'll also only be thinking about paths of Page views, but the thought process for custom events is exactly the same.
+
+![path visualization with only 2 steps](../../images/docs/user-guides/paths/2-step.png)
+
+Here, I've set the [start point](#changing-the-start-and-end-point) to be the Docs homepage for PostHog, and I've [limited the number of paths](#maximum-number-of-paths) to only 5.
+With this visualization, you can see that the core idea of paths is very simple: Each step along the path is a page that the user viewed, and the horizontal bars correspond to a user navigating from one page to another.
+The numbers next to the paths correspond to how many distinct users passed through this specific step along the way. So out of the 309 people who viewed the home page over this period, 99 of them continued on to `/docs/deploy` and so on.
+
+### Adding a 3rd step
+
+Now that we have an idea of how the 2-step path works, let's add a 3rd step so we can learn about drop off. As a disclaimer, I've filtered out a bunch of other user paths here to make things easier to understand.
+
+![path visualization with only 3 steps](../../images/docs/user-guides/paths/3-step.png)
+
+As you can see, we now have 3 steps in our journey, starting with a user viewing the `/docs/` page, and then continuing on to whatever page that user viewed next, with the 3rd step being wherever the user ended up.
+In this case, some users navigated directly to the `/docs/integrate` page, and as a result there is no intermediate step displayed.
+However for other users, such as those who first navigated to the `/docs/deploy` page, we can see a 3rd step now, corresponding to wherever they navigated after viewing the `/docs/deploy` page.
+
```

Comment:
> Note: If a user visits `/docs/deploy`, then refreshes the page, i.e. they visit `/docs/deploy` again, we count this as the same step. In other words, no two consecutive steps ever have the same event/URL.

Reply:
Definitely a good note to have, adding this in!